src/plugins/intel_gpu/src/kernel_selector/cl_kernels/sdpa_opt.cl

```diff
@@ -802,7 +804,7 @@ inline MASK_VECTOR_TYPE FUNC(load_attn_mask)(OPTIONAL_SHAPE_INFO_ARG
 #endif
 
     // Apply scale to attn_mask
-#if IS_CAUSAL || HAS_ATTN_MASK_INPUT
+#if IS_CAUSAL || HAS_ATTN_MASK_INPUT || defined(STATIC_SCALAR_ATTN_MASK_VALUE)
     mask_vec *= scale_val;
 #endif
```

Comment:
I think `mask_vec` should be initialized with `STATIC_SCALAR_ATTN_MASK_VALUE` before this scaling

Reply:
Updated, thanks!